scripts/build.sh

```diff
@@ -112,6 +112,8 @@ build() {
     mkdir -p ${BUILD_DIR}
     cd ${BUILD_DIR}
     cmake \
+        -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+        -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
```

Comment:
It looks like to conform to current behaviour there should be check for the `ccache` existence. And it would be nice to allow a user to specify a launcher, e.g. maybe somebody wants to use `distcc` or something custom.

Reply:
Ok, this is a reasonable request.

I won't be able to get to this today, but sometime this week, probably.